src/WebJobs.Extensions.DurableTask/ProviderUtils.cs

```diff
@@ -61,5 +61,18 @@ public static DurableOrchestrationStatus ConvertOrchestrationStateToStatus(Orche
         {
             return DurableClient.ConvertOrchestrationStateToStatus(orchestrationState);
         }
+
+        /// <summary>
+        /// Trace a warning event for the extension.
+        /// </summary>
+        /// <param name="extension">The durable task extension.</param>
+        /// <param name="hubName">The name of the task hub.</param>
+        /// <param name="functionName">The function name.</param>
+        /// <param name="instanceId">The instance id.</param>
+        /// <param name="message">The message.</param>
+        public static void TraceWarningEvent(this DurableTaskExtension extension, string hubName, string functionName, string instanceId, string message)
```

Comment:
Is this still being used or can it be reverted?

Reply:
I actually don't see anything in ProviderUtils.cs being used anywhere so I'm going to delete the file.